functions_task1.js

```diff
@@ -0,0 +1,9 @@
+function multiple(numberFirst, numberSecond) {
+    return numberFirst * numberSecond;
+};
+function add(mult, numberThree) {
```

Comment:
Here, it is better to name the arguments just 'first' and 'second', but not 'mult' and 'numberThree'. The problem is that we can use the function 'add' anywhere to sum any two numbers. But here, you are assuming that one argument is always the result of previous multiplication while this does not have to be always true